pom.xml

```diff
@@ -49,11 +49,11 @@
     <version.io.opentracing.contrib-java-jdbc>0.0.3</version.io.opentracing.contrib-java-jdbc>
     <version.io.github.openfeign-feign-okhttp>9.5.0</version.io.github.openfeign-feign-okhttp>
     <version.io.github.openfeign.opentracing>0.0.5</version.io.github.openfeign.opentracing>
+    <!-- spring-boot-starter-parent is a module of spring-boot-dependencies
+        https://github.com/spring-projects/spring-boot/blob/master/spring-boot-starters/spring-boot-starter-parent/pom.xml -->
     <version.org.springframework.boot>1.4.1.RELEASE</version.org.springframework.boot>
-    <version.org.springframework.cloud-spring-cloud-netflix>1.2.1.RELEASE</version.org.springframework.cloud-spring-cloud-netflix>
+    <version.org.springframework.cloud-spring-cloud-dependencies>Camden.SR7</version.org.springframework.cloud-spring-cloud-dependencies>
```

Comment:
should I override `version.org.springframework.cloud-spring-cloud-dependencies` property in my `pom.xml` if my app uses `Dalston`?

Reply:
thanks